spec/features/user_deletes_review_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature "user deletes an existing beach", %{
+  As an authenticated user
+  I want to delete a review that I wrote
+  So that no one can view it
+} do
+
+  scenario "\n delete beach from its show page" do
+    user = FactoryGirl.create(:user)
+    beach = FactoryGirl.create(:beach, user: user)
+    FactoryGirl.create(:review, beach: beach, user: user)
+
+    visit new_user_session_path
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+
+    click_button 'Log in'
```

Comment:
At some point, you probably want to write a helper method like `sign_in_as(user)` for lines 14-19 to dry this code up.